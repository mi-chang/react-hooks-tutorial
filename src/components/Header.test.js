import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';

describe('Header', () => {
    it('has text', () => {
        const { container, getByText } = render(<Header text='header text' />);
        expect(getByText('header text')).toBeInTheDocument();
        expect(container).toMatchSnapshot();
    })
});