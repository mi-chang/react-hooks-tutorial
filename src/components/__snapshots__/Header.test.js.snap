// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header has text 1`] = `
<div>
  <header
    class="App-header"
  >
    <h2>
      header text
    </h2>
  </header>
</div>
`;
